dspace/config/modules/versioning.cfg

```diff
@@ -3,6 +3,10 @@
 #---------------------------------------------------#
 #  These configs are used by the versioning system  #
 #---------------------------------------------------#
+# The property versioning.enabled is used to enabled/disable versioning in DSpace,
+# the default value is true if it unset
+# versioning.enabled=
```

Comment:
Tiny thing, usually we put the default value after the `=` sign.  So, we might want to make this say `# versioning.enabled=true`